conf/bridge-server.conf

```diff
@@ -1,7 +1,7 @@
 synapse.user = yours-synapse-user
 synapse.api.key = yours-synapse-api-key
 
-exporter.synapse.id = 3325672
+exporter.synapse.id = exporter-synapse-id
```

Comment:
Does this need to be updated in Heroku? Should this be part of this change or should we be including it in a separate change?